drivers/arm/pl011/aarch64/pl011_console.S

```diff
@@ -6,6 +6,7 @@
 #include <arch.h>
 #include <asm_macros.S>
 #include <assert_macros.S>
+#include <console_macros.S>
```

Comment:
Whoops, this must have been an oversight in the original multi-console patch. Thanks for the fix!